lib/generators/devise_remote/templates/add_ldap_attributes_template.erb

```diff
@@ -0,0 +1,27 @@
+class AddLdapAttributesTo<%= table_name.camelize %> < ActiveRecord::Migration<%= migration_version %>
+  def self.up
+    add_column :<%= table_name %>, :display_name, :string
```

Comment:
Are these fields needed/wanted for this gem?  Should groups be added if we do want them?

Reply:
@cam156 yeah, I'm not so sure either. I'm leaning towards taking them out...